### ISSUE TITLE
[0.9.5-rc3] No response for read queries after storing few megas of data for more than an hour

### ISSUE THREAD

**Issue by phagunbaya** [influxdb/influxdb#4844]
I'm experiencing this issue on ubuntu 14.04 OS with influx version 0.9.5-rc3.
Read queries for fetching measurement data does not send any response. Even "SHOW DATABASES" does not give any response.

Logs - 
```
[wal] 2015/11/19 14:51:39 write to index of partition 1 took 1.392237758s
[wal] 2015/11/19 14:51:47 Flush due to idle. Flushing 1104 series with 1104 points and 68106 bytes from partition 1
[wal] 2015/11/19 14:51:48 write to index of partition 1 took 1.494225044s
[wal] 2015/11/19 14:51:57 Flush due to idle. Flushing 1104 series with 1104 points and 68106 bytes from partition 1
[wal] 2015/11/19 14:51:58 write to index of partition 1 took 1.392084508s
[wal] 2015/11/19 14:52:06 Flush due to idle. Flushing 1104 series with 1104 points and 68106 bytes from partition 1
[wal] 2015/11/19 14:52:08 write to index of partition 1 took 1.538364629s
[wal] 2015/11/19 14:52:17 Flush due to idle. Flushing 1104 series with 1104 points and 68106 bytes from partition 1
[wal] 2015/11/19 14:52:18 write to index of partition 1 took 1.459369501s
[wal] 2015/11/19 14:52:26 Flush due to idle. Flushing 1104 series with 1104 points and 68106 bytes from partition 1
[wal] 2015/11/19 14:52:28 write to index of partition 1 took 1.261158044s
[wal] 2015/11/19 14:52:37 Flush due to idle. Flushing 1104 series with 1104 points and 68106 bytes from partition 1
[wal] 2015/11/19 14:52:38 write to index of partition 1 took 1.270582687s
[wal] 2015/11/19 14:52:47 Flush due to idle. Flushing 1104 series with 1104 points and 68106 bytes from partition 1
[wal] 2015/11/19 14:52:48 write to index of partition 1 took 1.314870213s
[wal] 2015/11/19 14:52:56 Flush due to idle. Flushing 1104 series with 1104 points and 68106 bytes from partition 1
[wal] 2015/11/19 14:52:58 write to index of partition 1 took 1.521373346s
[retention] 2015/11/19 14:53:01 retention policy shard deletion check commencing
[retention] 2015/11/19 14:53:01 retention policy enforcement check commencing
[wal] 2015/11/19 14:53:07 Flush due to idle. Flushing 1104 series with 1104 points and 68106 bytes from partition 1
[wal] 2015/11/19 14:53:08 write to index of partition 1 took 1.303651811s
[wal] 2015/11/19 14:53:17 Flush due to idle. Flushing 1104 series with 1104 points and 68106 bytes from partition 1
[wal] 2015/11/19 14:53:19 write to index of partition 1 took 1.747648962s
[wal] 2015/11/19 14:53:27 Flush due to idle. Flushing 1104 series with 1104 points and 68106 bytes from partition 1
[wal] 2015/11/19 14:53:28 write to index of partition 1 took 1.349422797s
[wal] 2015/11/19 14:53:37 Flush due to idle. Flushing 1104 series with 1104 points and 68106 bytes from partition 1
[wal] 2015/11/19 14:53:39 write to index of partition 1 took 1.625171561s
[wal] 2015/11/19 14:53:47 Flush due to idle. Flushing 1104 series with 1104 points and 68106 bytes from partition 1
[wal] 2015/11/19 14:53:48 write to index of partition 1 took 1.415504554s
[wal] 2015/11/19 14:53:56 Flush due to idle. Flushing 1104 series with 1104 points and 68106 bytes from partition 1
[wal] 2015/11/19 14:53:58 write to index of partition 1 took 1.895083s
[wal] 2015/11/19 14:54:07 Flush due to idle. Flushing 1104 series with 1104 points and 68106 bytes from partition 1
[wal] 2015/11/19 14:54:09 write to index of partition 1 took 2.081084323s
[wal] 2015/11/19 14:54:16 Flush due to idle. Flushing 1104 series with 1104 points and 68106 bytes from partition 1
[wal] 2015/11/19 14:54:18 write to index of partition 1 took 1.62580515s
```

**Comment by beckettsean** [comment]
@phagunbaya the logs are not indicative of anything, and if `SHOW DATABASES` is not returning, the instance is not healthy. Can you look in the logs for anything with `error` or ` 500 ` (http response codes)? What are the CPU, RAM, and IOPS usage on the machine? What happens if you restart the instance?

**Comment by phagunbaya** [comment]
Machine config:
ubuntu 14.04
CPU - 4
Memory - 8g

Influx daemon usage
CPU - 3%
Memory - 10.6%

IOPS
while writing data to influx
kB_read/s - 568.7
kB_wrtn/s - 376.9

idle state
kB_read/s - 3.47
kB_wrtn/s - 61.97

no logs with `error` and no `500` responses. The http API request (`http://192.168.1.222:8086/query?q=select+*+from+%22Pipeline%22&db=chu6mn0639aw9i`) stays pending.

**Comment by beckettsean** [comment]
The statistics timeout error is irrelevant, I believe.

Please share the entire log starting at least 5 minutes before the database stops responding.

What does `curl -iG 'http://192.168.1.222:8086/ping' return when the database is not responding to queries?

**Comment by phagunbaya** [comment]
curl -iG 'http://192.168.1.222:8086/ping'
```
Phaguns-MacBook-Pro: phagunbaya$ curl -iG 'http://192.168.1.222:8086/ping' 
HTTP/1.1 204 No Content
Request-Id: 2fe91735-9016-11e5-91d3-000000000000
X-Influxdb-Version: 0.9.5-rc3
Date: Sat, 21 Nov 2015 06:07:47 GMT
```

Now I'm noticing that the database which has less data started responding. But the one which has few megs of data is not responding to queries still.

Logs with `500` response code 
```
[http] 2015/11/19 09:53:41 192.168.1.33 - root [19/Nov/2015:09:53:36 +0000] POST /write?db=chu6mn0639aw9i&p=%5BREDACTED%5D&precision=ms&time_precision=ms&u=root HTTP/1.1 500 8 - - 66d6af75-8ea3-11e5-8098-000000000000 5.007433664s
[query] 2015/11/19 09:53:41 SHOW MEASUREMENTS
[http] 2015/11/19 09:53:41 192.168.1.33 - root [19/Nov/2015:09:53:36 +0000] POST /write?db=chu6mn0639aw9i&p=%5BREDACTED%5D&precision=ms&time_precision=ms&u=root HTTP/1.1 500 8 - - 66d10842-8ea3-11e5-8094-000000000000 5.056668193s
[http] 2015/11/19 09:53:41 192.168.1.33 - root [19/Nov/2015:09:53:36 +0000] POST /write?db=chu6mn0639aw9i&p=%5BREDACTED%5D&precision=ms&time_precision=ms&u=root HTTP/1.1 500 8 - - 66d27a13-8ea3-11e5-8095-000000000000 5.086741679s
[http] 2015/11/19 09:53:41 192.168.1.33 - root [19/Nov/2015:09:53:36 +0000] POST /write?db=chu6mn0639aw9i&p=%5BREDACTED%5D&precision=ms&time_precision=ms&u=root HTTP/1.1 500 8 - - 66d3db47-8ea3-11e5-8096-000000000000 5.079616407s
[http] 2015/11/19 09:53:41 192.168.1.33 - root [19/Nov/2015:09:53:36 +0000] POST /write?db=chu6mn0639aw9i&p=%5BREDACTED%5D&precision=ms&time_precision=ms&u=root HTTP/1.1 500 8 - - 66d54a36-8ea3-11e5-8097-000000000000 5.071190948s
```

Complete Logs - https://drive.google.com/file/d/0ByZSwkp_d2U-RzFmX0VZdWk0NU0/view?usp=sharing

**Comment by phagunbaya** [comment]
After restarting it second time this morning, it's responding to every query.

**Comment by beckettsean** [comment]
Likely the same as https://github.com/influxdb/influxdb/issues/4870

**Comment by beckettsean** [comment]
Believed fixed by https://github.com/influxdb/influxdb/pull/4913, which is available in the just-released version `0.9.5.1`.
